.changeset/giant-teachers-turn.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/theme': patch
+'@twilio-paste/core': patch
+---
+
+[Theme] Fix sizeIcon types to allow for customiztaion
```

Comment:
todo: typo in customization.

thought: this changelog message isn't very clear. Can we provide more context here so our downstream consumers can more easily determine the intent please?